sycl/include/sycl/ext/intel/experimental/esimd/memory.hpp

```diff
@@ -1984,14 +1999,15 @@ constexpr void check_lsc_block_2d_restrictions() {
 ///  N = roundUpNextMultiple(BlockHeight, 4 / sizeof(T)) *
 ///   getNextPowerOf2(BlockWidth) * NBlocks
 ///
-template <typename T, int BlockWidth, int BlockHeight = 1, int NBlocks = 1,
+template <typename Tx, int BlockWidth, int BlockHeight = 1, int NBlocks = 1,
```

Comment:
`Tx` is not documented in this comment description.
Rather than renaming `T` to `Tx`, can you please keep `T` for user's type,
and add the using below as `RawT`?